tests/unit/timeutil/test_s32.c

```diff
@@ -90,7 +90,7 @@ static const struct timeutil_test_data tests[] = {
 		  .tm_wday = 5,
 		  .tm_yday = 364,
 	  } },
-	{ .unix = 63072000,
```

Comment:
What is wrong with the name unix ?

Reply:
A lot of toolchains add `unix` as a predefined preprocessor symbol to indicate the host system type.  Ideally that should be inhibited by toolchain options (making it `__unix__` instead).  I assumed this was changed because of such a problem, though I don't like the change.

If that isn't the case, the name should be changed back, or it should be switched instead to `posix` or something that's a bit more indicative of how the value is to be interpreted.  `unix_timestamp` would also be a satisfactory substitute.

Reply:
Yes, "unix" is a predefined preprocessor symbol. We have to change it.